IDE/Renesas/e2studio/RX72N/EnvisionKit/Simple/test/test.scfg

```diff
@@ -722,125 +722,125 @@
             <allocatable id="TMR0">
                 <isocket id="CountingClock" selection="Clock.tool_clock_pclkb" value="6.0E7"/>
                 <option enabled="true" id="CountMode" selection="8bitMode">
-                    <item id="8bitMode" input="" vlaue="0"/>
-                    <item id="16bitMode" input="" vlaue="0"/>
+                    <item id="8bitMode" input="" value="0"/>
```

Comment:
Not sure you can/should fix these. This is a property for a code generation tool. Prefer you just add an exception for these files.